Describe item types (#84)
citation-style-language/documentation: specification.rst

    or other publication;
    Use for preprints, working papers, and similar works posted on a platform
    where some level of persistence or stewardship is expected (e.g. ArXiV or 
    other preprint repositories, working paper series);
    For unpublished works not made widely available or only hosted on personal
    websites, use ``manuscript``

article-magazine
    An article published in a non-academic magazine

article-newspaper
    An article published in a newspaper

article-journal
    An article published in an academic journal

bill
    A proposed piece of legislation

book
    a;
    If a ``container-title`` is present, the item is interpreted as a book
    republished in a collection or anthology

broadcast
    A recorded work broadcast over an electronic medium (e.g. a radio 
    broadcast, a television show, a podcast);
    If ``container-title`` is present, the item is interpreted as an episode
    contained within a larger broadcast series (e.g. an episode in a television
    show or an episode of a podcast)

chapter
    A part of a book cited separately from the book as a whole (e.g. a chapter

>     in an edited book); may also be used for introductions, forewards, and

Reviewer: ```suggestion
    in an edited book); may also be used for introductions, forewords, and
```

>     similar supplmental components of a book

Reviewer: ```suggestion
    similar supplemental components of a book
```

report
    A technical report, government report, white paper, brief, or similar
    work distributed by an institution; also used for manuals (e.g. a software, 
    instrument, or test manual);
    If a ``container-title`` is present, the item is interpreted as a chapter
    contained within a larger report

review
    A review of an item other than a book (e.g. a film review, posted peer 
    review of an article);
    If ``reviewed-title`` is absent, ``title` is taken to be the title of the 
    reviewed item

review-book
    A review of a book;
    If ``reviewed-title`` is absent, ``title` is taken to be the title of the 
    reviewed book

song
    An audio recording. 

speech
    A speech or other presentation (e.g. a paper, talk, poster, or symposium
    at a conference);
    Use ``genre`` to specify the type of presentation;
    Use ``event`` to indicate the event where the presentation was made (e.g.
    the conference name);
    Use ``container-titlee`` if the presentation is part of a larger session
    (e.g. a paper in a symposium);
    For papers published in conference proceedings, use ``paper-conference``;
    For artistic performance, use ``performance``

thesis

>     A thesis written for an educational institution to satisfy the requirements
>     for a degree;

Reviewer: ```suggestion
    A thesis written to satisfy the requirementsof a degree;
```

    Use ``genre`` to specify the type of thesis

treaty

>     A treaty agreement between two political authorities

Reviewer: ```suggestion
    A treaty agreement between political authorities
```

(obvious not all treaties are bilateral

Author: 🤦‍♂️

review
    A review of an item other than a book (e.g. a film review, posted peer 
    review of an article);
    If ``reviewed-title`` is absent, ``title` is taken to be the title of the 
    reviewed item

review-book
    A review of a book;
    If ``reviewed-title`` is absent, ``title` is taken to be the title of the 
    reviewed book

song
    An audio recording;
    Can be used for any audio recording (not only music);
    If a ``container-title`` is present, the item is interpreted as a track
    contained within a larger album or compilation of recordings

speech
    A speech or other presentation (e.g. a paper, talk, poster, or symposium
    at a conference);
    Use ``genre`` to specify the type of presentation;
    Use ``event`` to indicate the event where the presentation was made (e.g.
    the conference name);
    Use ``container-titlee`` if the presentation is part of a larger session
    (e.g. a paper in a symposium);
    For papers published in conference proceedings, use ``paper-conference``;
    For artistic performances, use ``performance``

thesis
    A thesis written to satisfy requirements for a degree;
    Use ``genre`` to specify the type of thesis

treaty

>     A treaty agreement between political authorities

Reviewer 2: ```suggestion
    Use ``genre`` to specify the type of thesis

treaty
    A treaty agreement among political authorities
```

    An entry in an encyclopedia or similar reference work

figure

>     A figure illustrating or presenting research data; may be in any format 

Reviewer 2: 1. "figure" is circular; no"; maybe "graphic"?
2. perhaps should strike "research"? that qualifier seems to make it too narrow?

Author: The major question is what is the difference between `figure` and `graphic`?

Author: Thinking this through

In terms of non-written material, we have three broad item types:
  1. `graphic`: still visual items
  2. `song`: audio items
  3. `motion_picture`: video items

Those cover the major citation needs, with `genre` and `medium` providing subtypes and format descriptions. 

But then there's `figure`. What exactly is that for? For example, a biology paper might contain a still image of a protein. That could be a `graphic`. A video demonstrating how the protein folds could be a `motion_picture`. But both of those could also be a `figure`. I think there is potential value in having a specific item type to refer to things like [this figure extracted from an article](https://figshare.com/articles/Disruption_of_lipid_rafts_reduces_I_sub_Na_sub_and_alters_the_impact_of_twin_NEPs_on_the_inward_current_/12455345) or [this figure we posted standalone](https://psyarxiv.com/bm28v/). I'm not too worried about their being conceptual overlap here. If an item can reasonably be described as a `figure` for research, then the type is appropriate?

Perhaps the description for figure should be:
`An illustration or representation of data;`
`May be in any format`

We could potentially say that `figure` specifically is a component extracted from a journal article, preprint, etc. (e.g., many medical journals assign individual DOIs to each figure and table so that they can be separately linked to and archived). I don't think we need to be that restrictive (e.g., the second item I linked to above).

Reviewer 3: I was just about to ask whether we need a figure type at all. Since I'd never cite the figure, just always the article that contains a figure. But in your last paragraph you say: 

> We could potentially say that figure specifically is a component extracted from a journal article, preprint, etc. (e.g., many medical journals assign individual DOIs to each figure and table so that they can be separately linked to and archived). I don't think we need to be that restrictive (e.g., the second item I linked to above).

Maybe that warrants for a table item type?

Author: One argument for `figure` existing is that sources like CrossRef and figshare have `figure` type and serve that as the item type. Assigning these to `figure` avoids the ambiguity of whether they should be `graphic`, `motion_picture`, etc. 

I'd say a table could go under `figure` if anything (that seems to be what CrossRef does).

    reviewed book

song
    An audio recording;

>     Can be used for any audio recording (not only music);

Reviewer 2: not sure about this. need it be a recording of a song performance (which is implicit here)?

Author: It's implemented in a general way in Zotero (as "Audio Recording"), and `song` seems to fill the same broad role for audio recordings as `motion_picture` does for video recordings. 

Thinking in terms of citation needs, diverse audio recordings generally have the same citation forms. Here, for example, are the example audio recording citation forms from the APA manual:
Bach, J. S. (2010). The Brandenburg concertos: Concertos BWV 1043 & 1060 [Album recorded by Academy of St Martin in the Fields]. Decca. (Original work published 1721)
Beethoven, L. van. (2012). Symphony No. 3 in E-flat major [Song recorded by Staatskapelle Dresden]. In Beethoven: Complete symphonies. Brilliant Classics. (Original work published 1804)
Beyoncé. (2016). Formation [Song]. In Lemonade. Parkwood; Columbia.
Bowie, D. (2016). Blackstar [Album]. Columbia.
Childish Gambino. (2018). This is America [Song]. mcDJ; RCA.
King, M. L., Jr. (1963, August 28). I have a dream [Speech audio recording]. American Rhetoric. https://www.americanrhetoric.com/speeches/mlkihaveadream.htm
Lamar, K. (2017). Humble [Song]. In Damn. Aftermath Entertainment; Interscope Records; Top Dawg Entertainment.

All of those can be handled easily with the existing `song` type. 

If `song` is specifically limited to musical recordings, then we would need to a new `audio` type or similar for poetry recordings, recordings of traffic noises, recordings of speeches. That doesn't seem necessary, and it also doesn't follow existing cataloguing practice--an ISRC number can refer to any type of audio or visual recording. 

I guess in sum, I don't think we have to be too caught up on the precise word used for type labels. `speech` is used more broadly than just speeches, for example, but to a variety of types of presentations. Even within music, a `song` might refer to an album, or a symphony,

    Use ``container-titlee`` if the presentation is part of a larger session
    (e.g. a paper in a symposium);
    For papers published in conference proceedings, use ``paper-conference``;
    For artistic performances, use ``performance``

thesis
    A thesis written to satisfy requirements for a degree;
    Use ``genre`` to specify the type of thesis

treaty

>     A treaty agreement between political authorities

Reviewer 2: strike "between" and replace with "among"

Author: FYI, you can propose changes like @adam3smith did. That would give me a button to click to just accept the proposal

Reviewer 2: Did what I just do result in that button?

Wasn't super intuitive.

Author: You made a direct change to the file (the ability for maintainers to do that is on by default in PRs in this repo). 

What Sebastian did was to "propose a change".
